pkg/networking/backend_sg_provider.go

```diff
@@ -345,17 +346,17 @@ func (p *defaultBackendSGProvider) getBackendSGName() string {
 }
 
 func isSecurityGroupDependencyViolationError(err error) bool {
-	var awsErr awserr.Error
-	if errors.As(err, &awsErr) {
-		return awsErr.Code() == "DependencyViolation"
+	var apiErr smithy.APIError
+	if errors.As(err, &apiErr) {
+		return apiErr.ErrorCode() == "DependencyViolation"
```

Comment:
Wondering if these error codes changed in v2?

Reply:
No these have not changed. I have verified it.